IonDotnet/Tree/Impl/IonTimestamp.cs

```diff
@@ -1,8 +1,8 @@
 using IonDotnet.Internals;
 
-namespace IonDotnet.Tree
+namespace IonDotnet.Tree.Impl
 {
-    public sealed class IonTimestamp : IonValue
+    public sealed class IonTimestamp : IonValue, IIonTimestamp

```

Comment:
Since we have a valuefactory, this class should be an internal class.

Reply:
Done.